services/user/InviteSys/ui/README.md

```diff
@@ -0,0 +1,68 @@
+# Fractally Webapp
```

Comment:
This isn't the Fractally Webapp, but rather InviteSys, right? Maybe I hadn't updated this when the app was in toolbox.